typhon/files/handlers/common.py

```diff
@@ -319,7 +330,8 @@ def __hash__(self):
     def __repr__(self):
         return f"FileInfo(\n  '{self.path}',\n" \
                f"  times={self.times},\n" \
-               f"  attr={self.attr}\n)"
+               f"  attr={self.attr},\n" \
+               f"  fs={self.file_system})"
```

Comment:
The ``__repr__`` method is used to represent an object in containers (e.g. a list of `FileInfo`) so this implementation might be a bit wordy. But this is just a fine-tuning comment that you can happily ignore.